app/models/measure_type.rb

```diff
@@ -28,10 +28,20 @@ def description
     translated_description || attributes['description']
   end
 
+  def description_only
+    attributes['description']
+  end
+
   def supplementary_unit_import_only?
     id.in?(SUPPLEMENTARY_IMPORT_ONLY_MEASURE_TYPES)
   end
 
+  def md_details
```

Comment:
```suggestion
  def details_text
```

Reply:
Not sure about abbreviations tbh. It feels weird that the client needs to know the format, too

Reply:
```suggestion
  def details_text
    Govspeak::Document.new(markdown_details_text, sanitize: true).to_html.html_safe
  end
```